SIAB/abacus/api.py

```
@@ -0,0 +1,436 @@
+    orbital_dir : str
+        where the generated orbital file is stored
+    
+    Returns
+    -------
+    dict
+        the atomspecies dictionary
+    '''
+    out = {'pp': os.path.abspath(pp)}
+    if all([ecut, rcut, lmaxmax]):
+        less_dof = 0 if primitive_type == 'normalized' else 1
+        nzeta = [_nbes(l, rcut, ecut) - less_dof for l in range(lmaxmax + 1)]
```

Ruff

Ambiguous variable name: `l`

(E741)